src/video/ngage/SDL_ngageevents.cpp

```diff
@@ -113,22 +113,22 @@ static SDL_Scancode ConvertScancode(_THIS, int key)
             keycode = SDLK_ASTERISK;
             break;
         case EStdKeyHash:         // Hash
-            keycode = SDLK_SLASH;
+            keycode = SDLK_HASH;
             break;
         case EStdKeyDevice0:      // Left softkey
-            keycode = SDLK_F1;
+            keycode = SDLK_LGUI;
```

Comment:
> You might want to use a different keycode here - SDL 1.2 maps EStdKeyLeftFunc and EStdKeyRightFunc to SDLK_LMETA and SDLK_RMETA (the old name for these keys).

> Any reason to maintain compatibility with SDL 1.2? I'm genuinely interested.

My main concern is that it may be desirable to add support for other S60v1/v2 devices that have `EStdKeyLeftFunc` and `EStdKeyRightFunc` keys, and it would be awkward to change the key mappings later once more applications are using the N-Gage port.

I had a go at adding new keycodes for the left and right softkeys, since Android has them as well: https://github.com/ccawley2011/SDL/commit/061b96f66924f43d363830729590e0afbe4d4abb. If you think it would be useful, I can open a PR with these changes.

Reply:
Sounds reasonable, please go ahead.

Reply:
OK, I've opened PR #5619.